index.php

```diff
@@ -41,6 +41,13 @@
         'img_url' => 'img/lot-6.jpg',
     ],
 ];
+
+function format_price (float $price): string
+{
+    $ceiled_price = ceil($price);
+    return $ceiled_price < 1000 ? ceil($ceiled_price) . ' ₽' : number_format($ceiled_price, 0, ',', ' ') . ' ₽';
+}
+
```

Comment:
У тебя идет повтор ceil  в условии `1000 ? ceil($ceiled_price)` это все равно что `1000 ? ceil(ceil($price))`
```suggestion
function format_price (float $price): string
{
    $ceiled_price = ceil($price);
    return $ceiled_price < 1000 ? ceil($price) . ' ₽' : number_format($ceiled_price, 0, ',', ' ') . ' ₽';
}

```